lib/sanbase_web/graphql/dataloader/postgres_dataloader.ex

```diff
@@ -132,4 +132,31 @@ defmodule SanbaseWeb.Graphql.MetricPostgresDataloader do
     |> Sanbase.Model.Project.List.by_slugs()
     |> Enum.into(%{}, fn %{slug: slug} = project -> {slug, project} end)
   end
+
+  def query(:social_volume_query, projects) do
+    all_projects = Project.List.projects()
+
+    Map.new(
+      projects,
+      fn project ->
```

Comment:
This implementation is ignoring the predefined custom queries.

This should be reworked like this:
```suggestion
    all_projects = Project.List.projects(preload: [:social_volume_query])
    
    Map.new(
      projects,
      fn
         %{social_volume_query: %{query: query}} -> query
         project ->       
```